Makefile

```diff
@@ -193,6 +193,7 @@ go_develop_and_test: go_develop go_test ## Generate protos, mocks and run all te
 
 # Inspired by @goldinguy_ in this post: https://goldin.io/blog/stop-using-todo ###
 # TODO                        - General Purpose catch-all.
+# TODO_COMMUNITY			  - A TODO that may be a candidate for outsourcing to the community.
```

Comment:
1. Indent seems off.
2. Make sure to also add it on line 218

Reply:
Ty!